pypeman/contrib/http.py

```diff
@@ -142,6 +142,7 @@ class HttpRequest(nodes.BaseNode):
         :param url: url to send.
         :param method: 'get', 'put' or 'post', use meta['method'] if None, Default to 'get'.
         :param headers: headers for request, use meta['headers'] if None.
+        :param cookies: cookies for request, use meta['cookies'] if None.
```

Comment:
shouldn't it use `meta.get("cookies", {})` or something
reading this doc string one could think, that we have a fail if `meta["cookies"]` doesn't exist